tracim/tracim/model/data.py

```diff
@@ -831,7 +1186,7 @@ def create_from_content_revision(cls, revision: ContentRevisionRO):
         action_description = ActionDescription(revision.revision_type)
 
         return VirtualEvent(id=revision.revision_id,
-                            created=revision.created,
+                            created=revision.updated,
```

Comment:
Before content view deletion ``revision.created`` and ``revision.updated`` columns was filled with revision creation date. Now, ``revision.created`` contain related content creation date.